log/operation_manager.go

```diff
@@ -158,18 +158,10 @@ func NewOperationManager(info OperationInfo, logOperation Operation) *OperationM
 
 // getActiveLogIDs returns IDs of logs eligible for sequencing.
 func (o *OperationManager) getActiveLogIDs(ctx context.Context) ([]int64, error) {
-	tx, err := o.info.Registry.LogStorage.Snapshot(ctx)
-	if err != nil {
-		return nil, fmt.Errorf("failed to create transaction: %v", err)
-	}
-	defer tx.Close()
-	logIDs, err := tx.GetActiveLogIDs(ctx)
+	logIDs, err := o.info.Registry.LogStorage.GetActiveLogIDs(ctx)
```

Comment:
This is so tantalisingly close to being just `return o.info.Registry.LogStorage.GetActiveLogIDs(ctx)`... 😃

Reply:
Inlined this invocation.